gulpfile.js

```diff
@@ -4,17 +4,19 @@ const gulp = require('gulp');
 const jasmine = require('gulp-jasmine');
 const istanbul = require('gulp-istanbul');
 const SpecReporter = require('jasmine-spec-reporter');
+const coveralls = require('gulp-coveralls');
+const IS_CI = process.env.CI || false;
```

Comment:
@yorkie hmm, Is it because set `NODE_ENV` let us determine the environment by manual?

I have update that in this commit https://github.com/weflex/semver-increment/pull/7/commits/8862716c50c9d305f25a45de21789f1dd9feabd2 .